apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx

```diff
@@ -45,8 +46,9 @@ function SectionProgressSelector({
   // If progress table is disabled, only show the v1 table.
   // If closed beta is disabled or the user is not in the closed beta, only show the v1 table.
   const isInClosedBeta =
-    DCDO.get('progress-table-v2-closed-beta-enabled', false) &&
-    progressTableV2ClosedBeta;
+    experiments.isEnabled(experiments.SECTION_PROGRESS_V2) ||
```

Comment:
Let's move this to `allowSelection` instead of `isInClosedBeta` just for clarity.